wls-gui-wahllokalsystem/src/components/common/dialogs/BaseDialogBegruendung.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <v-dialog
+    :model-value="visible"
+    persistent
+    width="auto"
+  >
+    <v-card>
+      <v-card-title>
+        <v-icon
+          icon="$warning"
```

Comment:
![grafik](https://github.com/user-attachments/assets/1a9ecb64-d35b-494e-8c5f-704cf225631a)
das triggert mich noch ein bisschen dass das icon nicht zentriert ist 
```suggestion
        <v-icon
          class="mb-1"
          icon="$warning"
```